gradle/libs.versions.toml

```diff
@@ -27,9 +27,12 @@ junit = "4.13.2"
 hilt = "2.42"
 kotlin = "1.7.0"
 lifecycle = "2.5.0"
+mapbox = "10.6.1"
```

Comment:
please update to `10.7.0` and remove `work-manager`

Reply:
Updated 10.6.1 -> 10.7.0
Removed workManager dependency